### PR TITLE
Remove php 7 non compatibility

### PR DESCRIPTION
**Pull request by adrian-stein**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "SilverStripe module to allow you to masquerade as other users",
     "type": "silverstripe-module",
     "require": {
-        "php": ">=5.3.3,<7",
+        "php": ">=5.3.3",
```
Comment (dhensby) [inline composer.json:6]: we can't/shouldn't really make judgements about whether this module will work on PHP8 so we should go with:

`"^5.3.3 || ^7"`

**Comment by dhensby** [comment]
OK - I've merged this by hand and updated the `composer.json` as well as the `.travis.yml`.

**Comment by adrian-stein** [comment]
Did the merge get completed? I am still seeing the `php:  ^7` on `v0.1.1`

**Comment by dhensby** [comment]
The merge has completed and is part of the `0.x-dev` branch, it's not been tagged yet (but I can do that).

v0.1.1 is a tag and won't change, I'll release v0.1.2 which will have the update, you'll need to update your constraint if you're locked onto a specific release.

**Comment by dhensby** [comment]
released v0.1.2: https://github.com/dhensby/silverstripe-masquerade/releases/tag/v0.1.2

**Pull request merged**